instrumentation/kubernetes-client-7.0/src/main/java/io/opentelemetry/instrumentation/auto/kubernetesclient/KubernetesClientTracer.java

```diff
@@ -55,6 +55,7 @@ protected String responseHeader(Response response, String name) {
 
   @Override
   protected Setter<Request> getSetter() {
+    // TODO (trask) no propagation implemented yet?
```

Comment:
Kubernetes client is a wrapper in okhttp instrumentation I think. We probably want to refactor this to inherit from the okhttp tracer at some point, assuming it's technical feasible

Reply:
Opened #1012